### PR TITLE
Add styleguides README.md

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: styleguides/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Styleguides
+
+Welcome to the Hack Club styleguide!
```
Comment (paked) [inline styleguides/README.md]: I don't like this sentence. I'd prefer something like the following:

> A styleguide is a document which tells us how to correctly format the different kinds of code in Hack Club repositories. We currently have them for:

**Comment by zachlatta** [comment]
Looks good to me.

We can do this in a separate pull request as well, but it may be worth adding links to the existing styleguides in this README.

**Comment by MaxWofford** [comment]
@zachlatta Am I missing any on lines 6-8?

**Comment by paked** [comment]
@zachlatta What do you mean by existing styleguides?

**Comment by zachlatta** [comment]
Oh, whoops. Totally my bad.

I misread the diff on my mobile client. All looks swell. I was referring to lines 6 - 8.

**Comment by paked** [comment]
Woohoo! YAY FOR REMOTE!

**Pull request merged**